src/bundle/ui-dev/src/modules/multi-file-upload/components/upload-popup/upload.popup.component.js

```diff
@@ -96,6 +96,7 @@ export default class UploadPopupModule extends Component {
             closeBtnAttrs: {
                 disabled: !itemsToUpload.length,
             },
+            ...((hasAnyUploadedItems || hasAnyItemsToUpload) && { extraClasses: 'c-tooltip-popup--with-uploaded-items' }),
```

Comment:
Personally I don't like this form, imho it's hard to read, I would do it as normal if below
```
if (hasAnyUploadedItems || hasAnyItemsToUpload) {
  tooltipAttrs.extraClasses = 'c-tooltip-popup--with-uploaded-items';
}
```